SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -2652,14 +2663,29 @@ UpdateDeletePage (
       //
       // Display GUID and help
       //
-      GuidToString (&Cert->SignatureOwner, GuidStr, 100);
-      GuidID = HiiSetString (PrivateData->HiiHandle, 0, GuidStr, NULL);
+
+      CertificateInfoStrLen = CertificateInfoStrSize;
+      if (
+        X509GetCommonName (
+            (UINT8*)Cert->SignatureData,
+            (UINTN)CertList->SignatureSize,
+            CertificateInfoStr8,
+            &CertificateInfoStrLen
+        ) == RETURN_SUCCESS

```

Comment:
Please use `Status = X509GetCommonName`... notation. Currently it is not well readable.

Reply:
[Fixed](https://github.com/Dasharo/edk2/pull/149/files#diff-49fdb815b38763cce7b30e2aea02c17111bc51206950ff1d21e0073ebd0cb1d3R2685)